### ISSUE TITLE
Problem with Extruder making Neutronium Large Plate

### ISSUE THREAD

**Issue by Fiyorin** [GTNewHorizons/GT-New-Horizons-Modpack#13391]
### Your GTNH Discord Username

Fiyorin

### Your Pack Version

2.3.3-nightly

### Your Server

private server

### Java Version

Java 19

### Type of Server

Vanilla Forge

### Your Expectation

i've been making this neutronium large plate in a industrial extruder with a tinkers large plate cast:

![image](https://user-images.githubusercontent.com/61357/235381891-5fd39db0-bc54-4dce-8a97-cbc409846245.png)

but since updating to this version it produces this instead:

![image](https://user-images.githubusercontent.com/61357/235381903-61de2997-195b-4f14-87e0-e268b613b920.png)

there's no recipe in NEI anymore. it should be 8 cosmic neutronium ingots per large plate.

### The Reality

.

### Your Proposal

.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by chochem** [comment]
![image](https://user-images.githubusercontent.com/40274384/235382479-297a6f8a-4935-4a43-a327-b264d0aee429.png)
this one has a recipe in NEI but you say you get the other so uh.

**Comment by Dream-Master** [comment]
One large plate can be made in avaritia table the other mods for in the extruder

**Comment by Fiyorin** [comment]
> ![image](https://user-images.githubusercontent.com/40274384/235382479-297a6f8a-4935-4a43-a327-b264d0aee429.png) this one has a recipe in NEI but you say you get the other so uh.

found the same recipe you showed but yea it still gives the wrong plate 
![image](https://user-images.githubusercontent.com/61357/235383515-d697980a-93e1-4055-90d7-da795b873339.png)

**Comment by Dream-Master** [comment]
The avaritia recipe give the plate with meta data 500 not 1671 . If not it is a bug

**Comment by Fiyorin** [comment]
maybe this caused and should be reverted? https://github.com/GTNewHorizons/TecTech/commit/22825e69eb1e0b06bcd9cccfd300b39b90a507a0

**Comment by Dream-Master** [comment]
> maybe this caused and should be reverted? [GTNewHorizons/TecTech@22825e6](https://github.com/GTNewHorizons/TecTech/commit/22825e69eb1e0b06bcd9cccfd300b39b90a507a0)

but why? you cant make the plates with id 500 or i am wrong ?

**Comment by Fiyorin** [comment]
> > maybe this caused and should be reverted? [GTNewHorizons/TecTech@22825e6](https://github.com/GTNewHorizons/TecTech/commit/22825e69eb1e0b06bcd9cccfd300b39b90a507a0)
> 
> but why? you cant make the plates with id 500 or i am wrong ?

no, there's no way to make the plate with id 500. neither extruder nor dire crafting table.

**Comment by Dream-Master** [comment]
but why you want this plate back ?
if you rervt this https://github.com/GTNewHorizons/TecTech/commit/22825e69eb1e0b06bcd9cccfd300b39b90a507a0 you cant use this recipe any more. I fix it

**Comment by Fiyorin** [comment]
> but why you want this plate back ? if you rervt this [GTNewHorizons/TecTech@22825e6](https://github.com/GTNewHorizons/TecTech/commit/22825e69eb1e0b06bcd9cccfd300b39b90a507a0) you cant use this recipe any more. I fix it

the plate is used in making the eye of harmony parts and the neutronium compressor, so it's needed
i don't know how it would change the recipe in that commit you'd revert
or you could add a new recipe in the avaritia crafting table to make the plate

**Comment by Fiyorin** [comment]
![image](https://user-images.githubusercontent.com/61357/235449099-71c4b3f4-064a-41e5-ad11-6e4403d958d4.png)

https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/commit/c4ca2b68e0baac9ead22a164b45a74ffa1454e49

**Comment by Fiyorin** [comment]
@Dream-Master upon further review: this still uses the neutronium plate (and not cosmic): 
![image](https://user-images.githubusercontent.com/61357/235449847-3926c4dd-e37f-4773-9074-11d048b5a8f4.png)

and there's still one plate left over on the neutronium compressor:
![image](https://user-images.githubusercontent.com/61357/235449982-ef821f24-5c18-4630-9c97-d90167d640bf.png)

**Comment by Dream-Master** [comment]
ok will fix in a min

**Comment by Dream-Master** [comment]
fixed
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/commit/2736c8af258b6e194d961c2e33e00eb42238d105
